docker/docker-compose.yml

```diff
@@ -0,0 +1,102 @@
+version: "3.9"
+services:

```

Comment:
this version is for compose version? please add a comment

Reply:
no,It is docker compose file version.
https://docs.docker.com/compose/compose-file/